src/QsCompiler/CompilationManager/TypeChecking.cs

```diff
@@ -425,8 +425,8 @@ internal static void ResolveGlobalSymbols(NamespaceManager symbols, List<Diagnos
             var declDiagnostics = symbols.ResolveAll(BuiltIn.NamespacesToAutoOpen);
```

Comment:
So before it was a position matched with the Range field of the QsCompilerDiagnostic, right? What did those to values before this change indicate? Now that it is just the Range field of the QsCompilerDiagnostic, has the conceptual definition of that field changed?

Reply:
Discussed offline: the Range field is always the position of the diagnostic relative to a given reference frame. The reference frame may change through the code, but users should always have enough context to know what reference frame is being used.